### ISSUE TITLE
Organize support for new features under TPC's raw command

### ISSUE THREAD

**Issue by jetrotal** [EasyRPG/Player#3034]
We discussed recently about running new features through TPC raw command:
![image](https://github.com/EasyRPG/Player/assets/45118493/98382b8d-c939-400f-88a6-81df4f76b891)
`@raw 11111, "string", 0, 1, 2, 3, 4, 5`

would be cool to know how to organize exclusive command IDs to finally implement new features to the Player

**Comment by Ghabry** [comment]
4000 - 4999 is reserved for EasyRPG commands. I already talked about this with BingShan years ago.

Though there is no logical pattern inside this area.

Not sure if there is any necessary though. 2k uses some grouping logic, 2k3 uses no logic (lol) and Cherry and BingShan use incrementing numbers.

-------

Well, feel free to request event numbers for allocation.

**Comment by jetrotal** [comment]
Posting this here, to not lose track on it:
![image](https://github.com/EasyRPG/Player/assets/45118493/f527fa1e-2544-43ef-8926-a4ff67cef3cd)
https://github.com/EasyRPG/Player/issues/2338

**Comment by jetrotal** [comment]
![image](https://github.com/EasyRPG/Player/assets/45118493/d6265fe5-83f3-4d3d-a904-42c05ba1c0db)
On rm2k, all multiline command IDs sarts with 2, instead of 1. is this relevant to us?

**Comment by Ghabry** [comment]
No, this is just the structure in the event commands of 2k but has no implications for us
